frontend/catalyst/compilation_pipelines.py

```diff
@@ -326,24 +320,12 @@ def _exec(shared_object_file, func_name, has_return, *args):
 
         setup(ctypes.c_int(argc), array_of_char_ptrs)
         function(*args)
-        teardown()
 
         result = args[0] if has_return else None
         retval = CompiledFunction.return_value_ptr_to_numpy(result) if result else None
 
-        if has_return:
-            raw_return = args[0].contents
-            for memref in raw_return:
-                is_constant = memref.allocated == 0xDEADBEEF
-                if is_constant:
-                    continue
-
-                if memref.allocated in numpy_managed_memory:
```

Comment:
I think this numpy managed memory is no longer needed right?

Reply:
You are right. Thanks for pointing it out. https://github.com/PennyLaneAI/catalyst/pull/78/commits/b01a049e99b6c442dd7f659fef3918c6721dadbc